diff --git a/autoload/cursorword.vim b/autoload/cursorword.vim
index 6b3f716..b0427ba 100644
--- a/autoload/cursorword.vim
+++ b/autoload/cursorword.vim
@@ -22,19 +22,25 @@ let s:alphabets = '^[\x00-\x7f\xb5\xc0-\xd6\xd8-\xf6\xf8-\u01bf\u01c4-\u02af\u03
 
 function! cursorword#matchadd(...) abort
   let enable = get(b:, 'cursorword', get(g:, 'cursorword', 1)) && !has('vim_starting')
-  if !enable && !get(w:, 'cursorword_match') | return | endif
+  if !enable && !get(w:, 'cursorword_match')
+    return
+  endif
   let i = (a:0 ? a:1 : mode() ==# 'i' || mode() ==# 'R') && col('.') > 1
   let line = getline('.')
   let linenr = line('.')
   let word = matchstr(line[:(col('.')-i-1)], '\k*$') . matchstr(line[(col('.')-i-1):], '^\k*')[1:]
-  if get(w:, 'cursorword_state', []) ==# [ linenr, word, enable ] | return | endif
+  if get(w:, 'cursorword_state', []) ==# [ linenr, word, enable ]
+    return
+  endif
   let w:cursorword_state = [ linenr, word, enable ]
   if get(w:, 'cursorword_match')
     silent! call matchdelete(w:cursorword_id0)
     silent! call matchdelete(w:cursorword_id1)
   endif
   let w:cursorword_match = 0
-  if !enable || word ==# '' || len(word) !=# strchars(word) && word !~# s:alphabets || len(word) > 1000 | return | endif
+  if !enable || word ==# '' || len(word) !=# strchars(word) && word !~# s:alphabets || len(word) > 1000
+    return
+  endif
   let pattern = '\<' . escape(word, '~"\.^$[]*') . '\>'
   let w:cursorword_id0 = matchadd('CursorWord0', pattern, -100)
   let w:cursorword_id1 = matchadd('CursorWord' . &l:cursorline, '\%' . linenr . 'l' . pattern, -100)
